packages/bento-frontend/src/pages/cart/cartView.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Grid, withStyles } from '@material-ui/core';
+import { TableView } from '../../bento-core/PaginationTable';
+import { configColumn } from './tableConfig/Column';
+import { themeConfig } from './tableConfig/Theme';
+import { Wrapper } from '../../bento-core/Wrapper';
+import { footerConfig } from './wrapperConfig/Wrapper';
+import { customTheme } from './wrapperConfig/Theme';
+import styles from './cartView.style';
+import Header from './cartHeader';
+
+const CartView = (props) => {
+  const {
+    classes,
+    config,
+    filesId = [],
+  } = props;
+
+  const Footer = () => (
+    <>
+      <Wrapper
+        wrapConfig={footerConfig}
+        customTheme={customTheme}
+        classes={classes}
+        section="myFiles"
+      />
+    </>
+  );
+
+  /**
+  * configure table state
+  */
+  const initTblState = (initailState) => ({
+    ...initailState,
```

Comment:
@rana22 Correct me if I am wrong, But Shouldn’t we spread initailState prop at the very end of the object?

Reply:
It should be declare before like redux state { ...state, updateValues } so default values can be updated.